src/backend/common/models.py

```diff
@@ -4,14 +4,27 @@
 from django.conf import settings
 from question.models import Solution
 
+
 class User(AbstractUser):
-    avatar = models.URLField(max_length=1024, blank=True, help_text="URL for avatar")
+    avatar = models.URLField(max_length=1024, blank=True,
+                             help_text="URL for avatar")
+    abilityId = models.ForeignKey(
```

Comment:
According to the naming convention of PEP8, use `ability_id`, `user_id`, and `question_id` instead.